scss/components/counter.scss

```diff
@@ -20,13 +20,16 @@ $block: #{$fd-namespace}-counter;
   @include fd-var-color("color", $fd-counter-color, --fd-counter-color);
   @include fd-var-color("background-color", $fd-counter-background, --fd-counter-background-color);
   font-size: $fd-counter-font-size;
+  font-family: $fd-font-family;
+  -webkit-font-smoothing: antialiased;
   line-height: 1;
   display: inline-block;
   text-align: center;
   padding: fd-space(1);
   &--notification {
+    font-size: $fd-counter-font-size;
```

Comment:
Why is there a font size declaration in here?  This is just a modifier class used in conjunction with the base class.